### PR TITLE
Release 4.5.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.5.1](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.5.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.5.1 (2021-02-11)
+
+- Fixed the ag-grdi wrapper to properly handle keyboard navigation while a master/detail grid is being used. [#98](https://github.com/blackbaud/skyux-ag-grid/pull/98)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:3]:
```suggestion
- Fixed the ag-grid wrapper to properly handle keyboard navigation while a master/detail grid is being used. [#98](https://github.com/blackbaud/skyux-ag-grid/pull/98)
```

**Review by blackbaud-paulah — CHANGES_REQUESTED** [review]
There was a minor typo in the changelog. I noted it in the suggestion.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.5.1](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.5.1)

**Review by blackbaud-paulah — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**